### ISSUE TITLE
Dashboard - Model Performance Technical Page

### ISSUE THREAD

**Issue by Duffew** [Duffew/mildew_detector#11]
As a **data practitioner** I want **to be able to see the technical details of the project** so that I can **assess the project's methodology**.

Acceptance Criteria:

Must:

The page provides performance metrics for::

- AC1 - Accuracy and validation accuracy in training
- AC2 - Loss and validation loss in training
- AC3 - Performance evaluation for test set
- AC5 - Number of images used

Should:

- SAC1 - Modal summary
